app/templates/index.hbs

```diff
@@ -1 +1 @@
-Index page works!
\ No newline at end of file
+<SelfClearCache @time="23 March 1:23 pm IST" @totalTimes="1"/>
```

Comment:
Once we have the API ready, we would be calling that API from `routes` file, pass that response in model
Get the response in controller for this template, mark it as `@tracked` (state variable) and then finally use that tracked variable here as props. 

If this sounds confusing, please read that again and refer this PR to get an idea of what I'm trying to say: https://github.com/Real-Dev-Squad/website-my/pull/82/files

I know we don't have the API yet, so we could create a mock object in controller file and get that mock data from controller to template, so we can remove the hardcoding and our way to integrating API (when it's ready) will become easier, less work to do then.

Let me know if there are any doubts :)